libs/ui/search/src/lib/results-table/results-table.component.html

```diff
@@ -117,7 +122,7 @@
       <span translate>record.metadata.updatedOn</span>
     </ng-template>
     <ng-template #cell let-item>
-      {{ dateToString(item.recordUpdated) }}
+      {{ isUnsavedDraft(item) ? '-' : dateToString(item.recordUpdated) }}
```

Comment:
I wished this column would be centered horizontally. I think that it would look nicer:
![image](https://github.com/user-attachments/assets/f4507eb2-2c4c-418a-a157-f0a2bc021bbd)

Not sure if it is necessary though. Let me know what you think.
All the cells use the same styling, so it would be a bit more specific to change this one. Just an idea of how you could center it without changing the other cells styling could be:

```    
    <ng-template #cell let-item>
      <div
        *ngIf="isUnsavedDraft(item); else date"
        class="flex justify-center w-full"
      >
        -
      </div>
      <ng-template #date
        ><div class="flex justify-center w-full">
          {{ dateToString(item.recordUpdated) }}
        </div></ng-template
      >
    </ng-template>
```